### ISSUE TITLE
Something wrong with debugging and inspecting, generally interacting with elements

### ISSUE THREAD

**Issue by aBabken** [ZFC-Digital/puppeteer-real-browser#241]
### Description

https://github.com/user-attachments/assets/ec0cf844-cb67-4196-96f9-d3df0e4e4113

In video I just try to interact with search bar, and something don't allow me fully make text, and this is with ALL elements to interact with every page

### Full steps to reproduce the issue

It's just default setting of puppeteer with last version
`
  "dependencies": {
    "@2captcha/captcha-solver": "^1.3.0",
    "axios": "^1.7.9",
    "date-fns": "^4.1.0",
    "dotenv": "^16.4.7",
    "js-levenshtein": "^1.1.6",
    "openai": "^4.76.1",
    "playwright": "^1.48.2",
    "playwright-extra": "^4.3.6",
    "puppeteer": "^23.8.0",
    "puppeteer-extra": "^3.3.6",
    "puppeteer-extra-plugin-click-and-wait": "^2.3.3",
    "puppeteer-extra-plugin-stealth": "^2.11.2",
    "puppeteer-real-browser": "^1.3.17",
    "puppeteer-screen-recorder": "^3.0.6"
  },
  "devDependencies": {
    "@cloudflare/puppeteer": "^0.0.14"
  }
`
It's not the full code, but the main part

`
const { connect } = require('puppeteer-real-browser');

    const { page, browser } = await connect({
        headless: false,
        turnstile: true,
        fingerprint: true,
        customConfig: {
            devtools: true
        },
        args: [
            '--no-sandbox',
            `--proxy-server=${randomProxy.ip}:${randomProxy.port}`
        ],
        proxy: {
            server: `${randomProxy.ip}:${randomProxy.port}`,
            username: randomProxy.username,
            password: randomProxy.password
        }
    })
    await page.setRequestInterception(true);

    page.on('request', (request) => {
        if (request.url().includes('screeb.app')) {
            request.abort();
        } else {
            request.continue();
        }
    });

    await page.setViewport(screenSize);

    await page.authenticate({
        username: randomProxy.username,
        password: randomProxy.password
    });`

### Issue Type

Bug

### Operating System

Mac OS

### Do you use Docker?

I don't use Docker

**Comment by RubberArchind** [comment]
it's because the turnstile, try set it to false.

**Comment by aBabken** [comment]
> turnstile

that's right, but I need turnstile also, is there any way to use it and avoid such beheviour?

**Comment by RubberArchind** [comment]
> > turnstile
> 
> that's right, but I need turnstile also, is there any way to use it and avoid such beheviour?

why you really need turnstile set to true, is that appear on every page or what?

**Comment by aBabken** [comment]
> > > turnstile
> > 
> > 
> > that's right, but I need turnstile also, is there any way to use it and avoid such beheviour?
> 
> why you really need turnstile set to true, is that appear on every page or what?

not on every page, just I want to except such case

**Comment by RubberArchind** [comment]
i think it's better to set it to false, and make the turnstile click "manually" based on some condition

**Comment by Nang-Maq** [comment]
Same here, I’m experiencing this on Facebook and Instagram too.  
It seems like the turnstile is the main issue.  
Elements are being clicked automatically in certain conditions, and it’s still happening to this day.

**Comment by aBabken** [comment]
@mdervisaygan it there any way to fix it to avoid automatic click?

**Comment by aBabken** [comment]
> i think it's better to set it to false, and make the turnstile click "manually" based on some condition

what do you mean "manually?

**Comment by mdervisaygan** [comment]
You may not send the turnstile key to turn off the turnstile click. However, if you want it to be resolved automatically, the library will continue with the current method. You can write your own click function.
